_includes/topnav.html

```diff
@@ -8,7 +8,7 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <a class="navbar-brand" href="{{site.baseurl}}/"><img src="{{site.baseurl}}/images/sun-small.png"><span class="projectTitle"> {{site.topnav_title}}</span></a>
+            <a class="navbar-brand" href="{{site.baseurl}}/"><img src="{{site.baseurl}}/images/sun-small.png" alt=""><span class="projectTitle"> {{site.topnav_title}}</span></a>
```

Comment:
alt="" is not a useful improvement.

It may satisfy some random linter, but doesn't actually improve accessibility or usability.

Reply:
ok thank you!